Form/DataTransformer/ObjectToIdTransformer.php

```diff
@@ -10,8 +10,8 @@
 use Symfony\Component\Form\Exception\InvalidConfigurationException;
 use Symfony\Component\Form\Exception\TransformationFailedException;
 
-class ObjectToIdTransformer implements DataTransformerInterface
-{
+class ObjectToIdTransformer implements DataTransformerInterface {
```

Comment:
According to [4.1. Extends and Implements (PSR-2)](http://www.php-fig.org/psr/psr-2/#extends-and-implements)
> The opening brace for the class MUST go on its own line;

please fix this